### PR TITLE
New package: invada-studio-plugins-0.3.1

### PR DESCRIPTION
**Pull request by laserbat**

### CODE REVIEW

File: srcpkgs/invada-studio-plugins/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+makedepends="ladspa-sdk"
+short_desc="Invada Studio LADSPA plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-2.0-or-later"
+homepage="https://launchpad.net/invada-studio"
+distfiles="https://launchpad.net/invada-studio/ladspa/0.3/+download/${pkgname}_${version}-nopkg.tar.gz"
+checksum=1f2566a5b27799f9fb2368acf8f447d813e7cd26465d6e9c428ca2ee5d37050a
+
+pre_configure() {
+    sed -i "s|/usr/local|${PKGDESTDIR}/usr|g" Makefile
```
Comment (maxice8) [inline srcpkgs/invada-studio-plugins/template]: can be DESTDIR

File: srcpkgs/invada-studio-plugins/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+do_build() {
+    make
```
Comment (maxice8) [inline srcpkgs/invada-studio-plugins/template]: please do make ${makejobs}

Comment (laserbat) [reply srcpkgs/invada-studio-plugins/template]: Done!

**Comment by laserbat** [comment]
Fixed the template

**Pull request merged**